diff -up mongodb-src-r2.2.3/SConstruct.debug mongodb-src-r2.2.3/SConstruct
--- mongodb-src-r2.2.3/SConstruct.debug	2013-05-08 14:47:37.438813257 +0200
+++ mongodb-src-r2.2.3/SConstruct	2013-05-08 21:59:14.483605789 +0200
@@ -520,6 +520,10 @@ elif os.sys.platform.startswith("linux")
     if force32:
         env.Append( EXTRALIBPATH=["/usr/lib32"] )
 
+    env.Append( EXTRALIBPATH=["__SCL_LIBDIR__"] )
+    env.Append( LINKFLAGS=["-L__SCL_LIBDIR__"] )
+    env.Append( LIBS=[ "unwind" ] )
+
     nix = True
 
     if static:
diff -up mongodb-src-r2.2.3/src/mongo/SConscript.debug mongodb-src-r2.2.3/src/mongo/SConscript
