diff -urN mongodb-src-r2.4.6.orig/SConstruct mongodb-src-r2.4.6/SConstruct
--- mongodb-src-r2.4.6.orig/SConstruct	2013-10-10 09:25:38.828712607 +0200
+++ mongodb-src-r2.4.6/SConstruct	2013-10-10 09:30:36.447305841 +0200
@@ -548,6 +548,8 @@
         env.Append( LINKFLAGS=" -static " )
     if has_option( "static-libstdc++" ):
         env.Append( LINKFLAGS=" -static-libstdc++ " )
+    if 'LD_LIBRARY_PATH' not in env['ENV']:
+        env['ENV']['LD_LIBRARY_PATH'] = os.environ['LD_LIBRARY_PATH']
 
 elif "sunos5" == os.sys.platform:
      nix = True
